website_lazy_load_image/tests/test_lazy_load_image.py

```diff
@@ -1,15 +1,17 @@
 # Copyright 2018 Onestein
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
-from odoo.tests.common import TransactionCase
+from odoo.tests.common import SavepointCase
 from lxml import etree
 
 
-class DateRangeTest(TransactionCase):
-    def setUp(self):
-        super().setUp()
+class DateRangeTest(SavepointCase):
```

Comment:
:thinking:  lemme guess... bad copy/paste session? :rofl:

Reply:
Changed it. More like lazy use of code snippets. :sweat_smile: